openproblems/tasks/label_projection/methods/xgboost.py

```diff
@@ -0,0 +1,73 @@
+from ....tools.decorators import method
+from ....tools.normalize import log_cpm
+from ....tools.normalize import log_scran_pooling
+from ....tools.utils import check_version
+from typing import Optional
+
+import functools
+import numpy as np
+
+_xgboost_method = functools.partial(
+    method,
+    paper_name="XGBoost: A Scalable Tree Boosting System",
+    paper_url="https://doi.org/10.1145/2939672.2939785",
+    paper_year=2016,
+    code_url="https://xgboost.readthedocs.io/en/stable/index.html",
+)
+
+
+def _xgboost(adata, test: bool = False, num_round: Optional[int] = None):
+    import xgboost as xgb
+
+    if test:
+        num_round = num_round or 2
+    else:  # pragma: nocover
+        num_round = num_round or 5
+
+    adata.strings_to_categoricals()
```

Comment:
isn't this now in the data decorator as standard?

Reply:
You're absolutely right